backend/src/nodes/utils/ncnn_model.py

```diff
@@ -174,14 +174,22 @@ def __init__(
         )
 
     def __getitem__(self, key: int) -> NcnnParam:
-        return self.param_dict[key]
+        try:
+            return self.param_dict[key]
+        except KeyError as e:
+            logger.error(f"Op {self.op} does not have param {key}, please report")
+            raise e
```

Comment:
This should be [just `raise`](https://docs.python.org/3/tutorial/errors.html#raising-exceptions). Otherwise, we lose the stack trace. The stack trace isn't hugely important here, but, you know, best practices.

```suggestion
            raise
```